examples/starrocks/deploy_a_starrocks_cluster_with_all_features.yaml

```diff
@@ -0,0 +1,174 @@
+# This manifest deploys a StarRocks cluster with all supported features.
```

Comment:
this is not enough, we still need a template with all possible configuration items, possibly deducting from the cr definition? that yaml is not friendly for human reading.